### PR TITLE
Fully Protected field

### PR DESCRIPTION
**Pull request by sgoertzen**

### CODE REVIEW

File: html/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <h1>Repoman - Github repository manager</h1>
     <table border="1">
         <thead>
-            <tr><th>Name</th><th>Protected</th></tr>
+            <tr><th>Name</th><th>Protected</th><th>Fully Protected</th></tr>
```
Comment (pdunnavant) [inline html/default.html:9]: 👎 Needs newlines.

Comment (sgoertzen) [inline html/default.html:9]: bite me

**Comment by pdunnavant** [comment]
👍

File: html/default.html (hunk 1 above)
Comment (pdunnavant) [inline html/default.html:9]: :fu:

**Pull request merged**